CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Guidelines for Contributing
+
+This project follows the [Apereo Contributing Guidelines](https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md).
```

Comment:
This link over-states the scope of `Jasig/uPortal/CONTRIBUTING.md`. At most `Jasig/uPortal/CONTRIBUTING.md` is uPortal-ecosystem contributing guidelines.